app/src/main/res/layout/item_video_two_columns.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    app:cardElevation="4dp"
+    android:layout_marginVertical="5dp"
+    android:layout_marginHorizontal="8dp"
+    android:foreground="?attr/selectableItemBackground">
+
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        >
+
+
+        <TextView
+            android:id="@+id/tvTitle"
+            android:layout_width="142dp"
+            android:layout_height="32dp"
+            android:layout_below="@+id/ivThumbnail"
+            android:layout_alignParentStart="true"
+            android:layout_marginStart="2dp"
+            android:layout_marginTop="0dp"
+            android:text="@string/warriors_at_celtics_full_game_6_nba_finals_highlights_june_16_2022"
+            android:textSize="9sp"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/tvChannelTitle"
+            android:layout_width="30dp"
+            android:layout_height="21dp"
+            android:layout_below="@+id/tvTitle"
+            android:layout_alignParentStart="true"
+            android:layout_marginStart="2dp"
+            android:layout_marginTop="0dp"
+            android:layout_toEndOf="@+id/tvTitle"
+            android:text="@string/abc_7_chicago"
+            android:textSize="8sp" />
+
+        <TextView
+            android:id="@+id/tvSeparator"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/tvTitle"
+            android:layout_marginStart="5dp"
+            android:layout_marginTop="0dp"
+            android:layout_toEndOf="@+id/tvChannelTitle"
+            android:text="#"
+            android:textSize="8sp" />
+
+        <TextView
+            android:id="@+id/tvPublishedAt"
+            android:layout_width="52dp"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/tvTitle"
+            android:layout_centerInParent="true"
+            android:layout_marginStart="5dp"
+            android:layout_marginTop="0dp"
+            android:layout_toEndOf="@+id/tvSeparator"
+            android:text="@string/_2022_07_04t17_00_12z"
+            android:textSize="8sp" />
+
+        <TextView
+            android:id="@+id/tvSeparator2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/tvTitle"
+            android:layout_marginStart="5dp"
+            android:layout_marginTop="0dp"
+            android:layout_toEndOf="@+id/tvPublishedAt"
+            android:text="#"
+            android:textSize="8sp"/>
```

Comment:
better to use Android Typography type
https://developer.android.com/reference/kotlin/androidx/compose/material/Typography